pkg/csi/service/vanilla/controller.go

```diff
@@ -264,6 +276,26 @@ func (c *controller) ReloadConfiguration(ctx context.Context) {
 	}
 }
 
+func filterIngoreDatastores(ctx context.Context, authMgr common.AuthorizationService, sharedDatastores []*cnsvsphere.DatastoreInfo) []*cnsvsphere.DatastoreInfo {
```

Comment:
This can be a private method in `controller`.

Reply:
Done.